src/app/layout.js

```diff
@@ -31,6 +31,7 @@ export default function RootLayout({ children }) {
       <link rel="icon" href="/images/favicon.svg" />
       <link rel="apple-touch-icon" href="/images/touch-icon.png" />
       <meta name="theme-color" content="#ffffff" />
+      <link href="https://fonts.googleapis.com/css2?family=Anton&display=swap" rel="stylesheet"/>
```

Comment:
Let's add Anton the same way we've added FormaDJR Banner.

- Download the font file from google
- Add a ref of it to the globals.css file
- Add local font to layout.js 
- Add too tailwind.config.js by replacing the heading key under fontFamily